conode/Makefile

```diff
@@ -29,20 +33,22 @@ newdb: conode
 	echo; echo "Shutting down conodes"; pkill -x conode; rm -f conodes/running
 
 # Use this target to build from local source
-docker: build/Dockerfile conode
+docker: build/Dockerfile cothority-pull
 	@export GO111MODULE=on GOOS=linux GOARCH=amd64; \
-	go build -o build/conode; \
-	cd ../cothority && \
-		go build -o ../conode/build/bcadmin ./byzcoin/bcadmin && \
-		go build -o ../conode/build/csadmin ./calypso/csadmin
+	go build -o build/conode
 	docker build -t $(IMAGE_NAME):$(TAG) -f build/Dockerfile build
 	docker tag $(IMAGE_NAME):$(TAG) $(IMAGE_NAME):dev
 
-docker_run: docker
+update_assets:
```

Comment:
Extractable, also, why stopping the docker in this target?

```make
$(WEPAPP_ASSET): variables.txt build/conodes/public.toml
        echo "ByzCoinID = \"$$( grep ByzCoinID $< | sed -e 's/.* //' )\"" > $@
        echo >> $@
        cat $(lastword $^) >> $@
```

Reply:
looks good, thanks.